zenpy/lib/api.py

```diff
@@ -1055,6 +1066,57 @@ def apply(self, macro):
 
         return self._query_zendesk(self.endpoint.apply, 'result', id=macro)
 
+    @extract_id(Macro)
+    def attachments(self, macro):
+        """
+        Shows attachments for a macro.
+        Zendesk API `Reference <https://developer.zendesk.com/rest_api/docs/support/macros#create-macro-attachment>`__.
+        :param macro: Macro object or id.
+        """
+
+        return self._query_zendesk(self.endpoint.attachments, 'macro_attachment', id=macro)
+        
+    def upload_attachment(self, fp, api_object="unassociated_macro", target_name=None):
```

Comment:
The param `api_object` here is a bit misleading as it would not accept any `api_object` right? It would be better IMO to have a boolean flag here `unassociated ` instead. If the flag is set then it is not associated with a macro. Is  bit easier for consumers of this API

Reply:
Agreed, it will be easier to understanding if there is a boolean here.